src/modules/layout/components/Header.js

```diff
@@ -103,7 +107,8 @@ class Header extends React.PureComponent<PropsType> {
   }
 
   render () {
-    const { theme } = this.props
+    const { theme } = this.props // TODO: When mering the share button task make sure to remove the mock t function
+    const t = (key: string) => key
```

Comment:
*reminder*

Reply:
You were faster! Pfff!

Reply:
Didn't even review your changes to the share task! :D